src/training/data.py

```diff
@@ -73,8 +72,8 @@ def set_epoch(self, epoch):
 
 
 def get_dataset_size(shards):
-    shards_list = list(braceexpand.braceexpand(shards))
-    dir_path = os.path.dirname(shards)
+    shards_list = wds.shardlists.expand_urls(shards)
```

Comment:
???

Reply:
uhh I have no idea why this was changed whoops

Reply:
I just realized it's from the first couple of commits from the OpenCLIP authors. Don't worry about it.